entity-framework/core/modeling/shadow-properties.md

```diff
@@ -44,11 +46,19 @@ var blogs = context.Blogs
 
 Shadow properties cannot be accessed after a no-tracking query since the entities returned are not tracked by the change tracker.
 
+## Configuring indexer properties
+
+You can use the Fluent API to configure indexer properties. Once you've called the string `IndexerProperty`, you can chain any of the configuration calls you would for other properties. In the following sample, `Blog` has indexer defined and it will be used to create indexer property.
```

Comment:
```suggestion
You can use the Fluent API to configure indexer properties. Once you've called the method `IndexerProperty`, you can chain any of the configuration calls you would for other properties. In the following sample, `Blog` has an indexer defined and it will be used to create an indexer property.
```